hofund-core/src/main/java/dev/logchange/hofund/connection/RequestMethod.java

```diff
@@ -0,0 +1,20 @@
+package dev.logchange.hofund.connection;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum RequestMethod {
```

Comment:
Nie lepiej użyć https://docs.spring.io/spring-framework/docs/4.0.0.M1_to_4.2.0.M2/Spring%20Framework%204.2.0.M2/org/springframework/http/HttpMethod.html

Reply:
@Athi unfortunately in hofund-core there is no spring dependency to allow using it f.e. in legacy java-ee applications (glassfish, wildfly, weblogic)